arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1953,11 +1624,31 @@
       },
       "allOf": [
         {
-          "$ref": "#/definitions/Resource"
+          "$ref": "#/definitions/TrackedResource"
         }
       ],
       "description": "Description of a namespace resource."
     },
+    "NamespacePatch": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "The sku of the created namespace"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NamespaceProperties"
```

Comment:
Why namespace PATCH model (NamespacePatch) includes `NamespaceProperties`? All the properties of `NamespaceProperties` [provisioningState, createdAt, updatedAt, serviceBusEndpoint and metricId] are read-only.